packages/e2e-tests/resolve-exports-svelte/vite.config.js

```diff
@@ -3,5 +3,10 @@ import { svelte } from '@sveltejs/vite-plugin-svelte';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-	plugins: [svelte({ compilerOptions: { css: 'none' } })]
+	plugins: [
+		svelte({
+			compilerOptions: { css: 'none' },
+			experimental: { disableSvelteResolveWarnings: false }
```

Comment:
I think we can remove this line since it's false by default
```suggestion
			compilerOptions: { css: 'none' }
```

Reply:
oops, this was from my local testing, keeping it there for easier switching wont hurt. its an e2e test app, so not affecting users in any way

Reply:
it confused me a bit when I first saw it because it made me think the default might be `true`, which is why I suggested potentially removing it